js/sparkswap.js

```diff
@@ -271,7 +271,46 @@ module.exports = class sparkswap extends Exchange {
     }
 
     async fetchOrder (id, symbol = undefined, params = {}) {
-        return this.privateGetV1OrderId ({ id });
+        const order = await this.privateGetV1OrdersId ({ id });
+        let status = 'open';
+        if (order.status === 'CANCELLED') {
+            status = 'cancelled';
+        }
+        if (order.status === 'COMPLETED') {
+            status = 'closed';
+        }
+        let type = 'limit';
+        if (order.is_market_order) {
+            type = 'market';
+        }
+        let side = 'buy';
+        if (order.side === 'ASK') {
+            side = 'sell';
+        }
+        const millisecondTimestamp = this.nanoToMillisecondTimestamp (order.timestamp);
+        const millisecondDatetime = this.nanoToMillisecondDatetime (order.datetime);
+        const response = {
+            'id': id,
+            'datetime': millisecondDatetime,
+            'timestamp': millisecondTimestamp,
+            'status': status,
+            'symbol': order.market,
+            'type': type,
+            'side': side,
+            'price': (this.safeFloat (order, 'amount') * this.safeFloat (order, 'limit_price')).toFixed (16),
+            'amount': this.safeFloat (order, 'amount'),
+            'filled': this.safeFloat (order, 'fill_amount') || 0,
+            'cost': this.safeFloat (order, 'fill_amount') * this.safeFloat (order, 'price') || 0,
+            'trades': [],
+            'info': order,
+        };
+        for (let i = 0; i < order.open_orders.length; i++) {
```

Comment:
When there are additional fields like this, I think it helps to link the format for this response at the top of the method.

Reply:
Do we do this for other methods?